apps/right-to-rent-check/behaviours/tenants.js

```diff
@@ -12,31 +13,106 @@ module.exports = fields => {
           return callback(err);
         }
 
-        // Find all legit fields
-        const picked = _.pick(values, fields);
-        // Get the list of tenants
-        const tenants = req.sessionModel.get('tenants') || [];
-        // Find a tenant matching the current tenant
-        let tenant = _.find(tenants, picked);
-
-        // Add a new tenant
-        if (!tenant) {
-          tenant = _.cloneDeep(picked);
-          if (!_.isEmpty(tenant)) {
-            tenants.push(tenant);
+        let tenant = _.pick(values, fields);
```

Comment:
I don't think we should annotate where there is already documentation online: https://lodash.com/docs/4.17.4#pick